loadgen/loadgen.cc

```diff
@@ -1353,9 +1354,28 @@ void RunPerformanceMode(SystemUnderTest* sut, QuerySampleLibrary* qsl,
   const LoadableSampleSet& performance_set = loadable_sets.front();
   LoadSamplesToRam(qsl, performance_set.set);
 
+  auto start_ts = PerfClock::now();
```

Comment:
Can you sample the start and end using both the PerfClock and the system_clock.

There are some platforms where the clock sources are different, and I think to_time_t will either fail to compile, hit a runtime error, or potentially fail silently.

Print the to_time_t using the system_clock. And print the duration of both the PerfClock and system_clock.

As a bonus:
1) print the ratio between the PerfClock and system_clock
2) Log an error if the ratio is more than 1% off and a warning if 0.1% off (even if print_timestamps is false.)